pgfinder/pgio.py

```diff
@@ -288,81 +288,31 @@ def read_yaml(filename: Union[str, Path]) -> Dict:
             return {}
 
 
-def long_to_wide(
+def pick_most_likely_structures(
```

Comment:
What is the logic for moving the matching to the `pgio` submodule?

This replaces `determine_most_likely_structure()` so it feels like it belongs in the `matching` sub-module.

If the `long_to_wide()` function is no longer required that is great, makes life considerably easier.

Reply:
I agree this would belong better in the matching submodule! It was just in pgio because it was replacing the `long_to_wide()` function. I'll move it now!